src/calculateRentalCost.js

```diff
@@ -28,7 +28,24 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRentCost = 40;
```

Comment:
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md Consider using the checklist for fixing constants naming